README.md

```diff
@@ -104,8 +104,8 @@ OPTIONAL -- Use with CAUTION:
 ## Development Setup
 
 Prerequisites:
-  - Python 3.7+, ideally installed via a virtual environment such as `pyenv`
-  - A local copy of the code
+  - Python 3.7+, ideally installed via a virtual environment. We recommend using [pyenv](https://github.com/pyenv/pyenv) for managing multiple Python installations with different versions, in conjunction with a tool like [pyenv-virtualenv](https://github.com/pyenv/pyenv-virtualenv) or [Python's venv module](https://docs.python.org/3.7/library/venv.html) for creating an isolated virtual environment.
```

Comment:
The reason I didn't do that is because in a similar discussion a month or so ago, Patrick seemed to allude that we should not be too opinionated on which virtual environment tool is used, as long as there's a virtual environment. So here I was just trying to make the reader aware that they can use either. Also, in the description of this ticket, there was a line about trying to make things "more similar to harmony-py" -- which offers both venv and pyenv-virtualenv as viable tools. Although this alone doesn't make this repo identical to harmony-py, it seemed like a step in that direction, without being too disruptive. I'm ok with either route and think you have a good point -- just explaining my thought process.

Reply:
I was mostly suggesting that to avoid complicating the end-user documentation, i.e. not mentioning or requiring one as a setup step.  It feels akin to telling people which editor to use (though I think we may also do that in other docs...)